### DISCUSSION TITLE
V3: JS error "attr is not defined" when redirecting

### DISCUSSION THREAD

**Discussion by renky** [livewire/livewire#5842] [7. Bugs (archived)]
We integrated v3 and Alpine via 
@livewireScriptConfig 
and 
import { Livewire, Alpine } from '../../vendor/livewire/livewire/dist/livewire.esm';
Livewire.start()

when doing a redirect from php as return-value we result in a JS-Error and redirecting is blocked

![image](https://github.com/livewire/livewire/assets/6528960/a157055a-825c-4291-bf3b-cade57f679fe)
![image](https://github.com/livewire/livewire/assets/6528960/b356e947-e885-42c1-a840-637586b3dc08)

The problematic script-section indeed doesn't have attributes:

```
<script>
              document.addEventListener("livewire:init", function(event) {
                   .... do something
              });
</script>
```

after removing all script-tags or adding type to it, redirecting works again -> unfortunatelly this is not a solution, debugbar for example also uses one without attributes.

**Comment by WyattCast44** [comment]
Additional Context/References: 
- https://github.com/alpinejs/alpine/issues/3670
- https://github.com/alpinejs/alpine/discussions/3671
- https://github.com/livewire/livewire/discussions/5829

**Comment by joshhanley** [comment]
@renky thanks for reporting. Yeah currently there is an issue where navigate is running even when it shouldn't be. See this failing test PR for more details https://github.com/livewire/livewire/pull/5859

**Comment by joshhanley** [comment]
This has been fixed in https://github.com/alpinejs/alpine/pull/3685 and should be in the next beta release.
